Packages/com.unity.inputsystem/InputSystem/Actions/Composites/ButtonWithOneModifier.cs

```diff
@@ -79,14 +79,16 @@ public class ButtonWithOneModifier : InputBindingComposite<float>
         /// still trigger. Default is false.
         /// </summary>
         /// <remarks>
-        /// By default, <see cref="modifier"/> is required to be in pressed state before or at the same time that <see cref="button"/>
+        /// By default, if the setting <see cref="InputSettings.shortcutKeysConsumeInput"/> is enabled,
+        /// <see cref="modifier"/> is required to be in pressed state before or at the same time that <see cref="button"/>
         /// goes into pressed state for the composite as a whole to trigger. This means that binding to, for example, <c>Shift+B</c>,
         /// the <c>shift</c> key has to be pressed before pressing the <c>B</c> key. This is the behavior usually expected with
         /// keyboard shortcuts.
         ///
         /// This parameter can be used to bypass this behavior and allow any timing between <see cref="modifier"/> and <see cref="button"/>.
         /// The only requirement is for them both to concurrently be in pressed state.
         /// </remarks>
+        [Tooltip("If checked, it will bypass the InputSettings.shortcutKeysConsumeInput setting and the modifier can be pressed after and the composite will still trigger")]
```

Comment:
Is that correct? It sounds like the reverse logic to my reading of the variable.

I think this is the state:

// Ordering important
overrideModifiersNeedToBePressedFirst = true 

// Ordering not important
overrideModifiersNeedToBePressedFirst = false


If that's not the case then the variable and text next to the checkbox would also need changing.

Reply:
Current state is :
if InputSettings.shortcutKeysConsumeInput is false (the default)
// Ordering not important in all case

if InputSettings.shortcutKeysConsumeInput is true
// Ordering important 
overrideModifiersNeedToBePressedFirst = false

// Ordering not important
overrideModifiersNeedToBePressedFirst = true

the name **override**ModifiersNeedToBePressedFirst override the setting and doesn't really describe the composite behavior.

Reply:
Wow that's confusing. Making it clearer with the tooltip is helpful but I'd prefer if we tidied this up further to make it much more intuitive.